### ISSUE TITLE
suggestion: consider adding this to go-mode

### ISSUE THREAD

**Issue by kpurdon** [syohex/emacs-go-add-tags#1]
Instead of providing this as a separate package would you consider adding this to the "official" `go-mode`? 

https://github.com/dominikh/go-mode.el

I have a dream of `go-mode` someday being on the level of https://github.com/jorgenschaefer/elpy . Adding useful things like this package to the root project would be a good start.

**Comment by syohex** [comment]
@dominikh  How about this ?

**Comment by dominikh** [comment]
> I have a dream of go-mode someday being on the level of https://github.com/jorgenschaefer/elpy .

Personally, I believe in a modular ecosystem, where features like this one live in their own packages, possibly reusing go-mode functionality. I don't want to maintain every single utility, and I don't want to force everyone into contributing to go-mode itself. 

Additionally, go-mode is avoiding third-party dependencies like `s` at the moment, so the code of this package couldn't be merged as is, anyway.

Maybe the go-mode repo should have a wiki page, however, where people can add their own packages.

tl;dr: I think this is fine where it is right now.

**Comment by syohex** [comment]
Thanks for comment.
